bindings_ffi/src/mls.rs

```diff
@@ -7975,10 +7976,10 @@ mod tests {
             .find_messages(FfiListMessagesOptions::default())
             .await
             .unwrap();
-        assert_eq!(bo_messages.len(), 3, "Bo should see 3 messages after sync");
+        assert_eq!(bo_messages.len(), 4, "Bo should see 3 messages after sync");
         assert_eq!(
             alix_messages.len(),
-            3,
+            4,
             "Alix should see 3 messages after sync"
         );
```

Comment:
I think this is still a bug we shouldn't be seeing multiple group update messages in stitched dms.